Signal/src/ViewControllers/ConversationView/ConversationInputToolbar.m

```diff
@@ -68,18 +70,34 @@ - (void)dealloc
     [self removeKVOObservers];
 }
 
+- (CGSize)intrinsicContentSize
+{
+    CGSize newSize = CGSizeMake(self.bounds.size.width, self.toolbarHeight + ConversationInputToolbarBorderViewHeight);
+    return newSize;
+}
+
+- (void)setToolbarHeight:(CGFloat)toolbarHeight
```

Comment:
Why bother with this method?

Reply:
Whoops - it used to do more, but it's no loner necessary. Removed, thanks!